src/engine/bufferscalers/enginebufferscalest.cpp

```diff
@@ -24,22 +24,20 @@ constexpr SINT kSeekOffsetFramesV20101 = 429;
 // TODO() Compensate that. This is probably cause by the delayed adoption of pitch changes due
 // to the SoundTouch chunk size.
 
-constexpr SINT kBackBufferSize = 1024;
+constexpr SINT kBackBufferFrameSize = 512;
```

Comment:
Why this change?

Reply:
This used to be premultiplied by 2 (line 34, used for back buffer allocation). Now it gets multiplied by the effective number of channel at runtime (line 93). In practice, the buffer remains of the same size for stereo channels, but will be extended for stem channels

Reply:
Make sense!